superbench/benchmarks/result.py

```diff
@@ -64,9 +65,22 @@ def add_raw_data(self, metric, value):
             )
             return False
 
-        if metric not in self.__raw_data:
-            self.__raw_data[metric] = list()
-        self.__raw_data[metric].append(value)
+        export_raw_data = 1
+        try:
+            export_raw_data = int(os.getenv('EXPORT_RAW_DATA', '1'))
+        except Exception as e:
+            logger.error(
+                'EXPORT_RAW_DATA is invalid: {}'.format(os.getenv('EXPORT_RAW_DATA', '1'))
+            )
+
+        if export_raw_data != 0:
+            if metric not in self.__raw_data:
+                self.__raw_data[metric] = list()
+            self.__raw_data[metric].append(value)
+        else:
+            with open(os.path.join(os.getcwd(), 'rawdata.log'), 'a') as f:
```

Comment:
the logic is opposite here? when `export_raw_data != 0` export/save it to log file?

Reply:
`export` means exporting to the caller. If save to log, we can use save_raw_data.

Reply:
If so, we export_raw_data is related to rawdata.log?

Reply:
Didn't get your point, there have two thing to determine,
1. the variable name, export_raw_data or save_raw_data?

which one do you prefer?
```
if export_raw_data:
    export to caller
else:
    save to file
```

or

```
if save_raw_data:
    save to file
else:
    export to caller
```

2. the default value, is 'export raw data to caller' or 'save the raw data to file'?

Currently, I used export_raw_data, and the default value is set as 1 (true) to export the raw data to caller.

Reply:
Solved